libs/auth/src/selectors/register.selector.ts

```diff
@@ -0,0 +1,43 @@
+import { createSelector, MemoizedSelector } from '@ngrx/store';
+
+import { selectAuthFeatureState } from './auth-feature.selector';
+import { DaffAuthToken } from '../models/auth-token';
+import {
+  DaffAuthRegisterReducerState,
+  DaffAuthFeatureState
+} from '../reducers/public_api';
+
+export interface DaffAuthRegisterSelectors<T extends DaffAuthToken> {
```

Comment:
The `T extends DaffAuthToken` looks like it isn't needed

Reply:
removed the unneeded generic